http/util.ts

```diff
@@ -4,7 +4,10 @@
 import { Status, STATUS_TEXT } from "./http_status.ts";
 import { deepMerge } from "../collections/deep_merge.ts";
 
-/** Returns true if the etags match. Weak etag comparisons are handled. */
+/** Returns true if the etags match. Weak etag comparisons are handled.
+ *
+ * @deprecated (will be removed in 0.184.0) use `etag/ifMatch` and `etag/ifNoneMatch` instead.
```

Comment:
Let's deprecate at v0.187.0
```suggestion
 * @deprecated (will be removed after 0.187.0) use `etag/ifMatch` and `etag/ifNoneMatch` instead.
```